tools/codegen/api/python.py

```diff
@@ -490,6 +490,8 @@ def argument_type_str(t: Type, *, simple_type: bool = False) -> str:
             return f'IntArrayRef[{size}]' if size is not None else 'IntArrayRef'
         elif str(t.elem) == 'Tensor':
             return f'TensorList[{size}]' if size is not None else 'TensorList'
+        elif str(t.elem) == 'Scalar':
+            return f'ScalarList[{size}]' if size is not None else 'ScalarList'
```

Comment:
Looks like it's possible to have things like 'ScalarList[3]'?

Reply:
Based on my meagre understanding of what list functions are to be used for, this is incredibly unlikely